tests/test_augmentation.py

```diff
@@ -461,14 +461,21 @@ def test_resize(self):
         assert results['gt_img'].shape[:2] == (128, 128)
 
         # test input with shape (256, 256)
-        results = dict(gt_img=self.results['img'][..., 0].copy())
-        resize = Resize(['gt_img'], scale=(128, 128), keep_ratio=False)
+        results = dict(gt_img=self.results['img'][..., 0].copy(), alpha=alpha)
+        resize = Resize(['gt_img', 'alpha'],
+                        scale=(128, 128),
```

Comment:
I thought what you want is a relative-scaling? Why not test it?

Reply:
mmcv.imrescale supports scale (float | Tuple[int]), so we don't need relative-scaling.

Reply:
Okay.